File: Bookcase core/src/main/java/cz/vhromada/book/repository/CategoryRepository.java
package cz.vhromada.book.repository;

import cz.vhromada.book.domain.Category;

import org.springframework.data.jpa.repository.JpaRepository;

/**
 * An interface represents repository for categories.
 *
 * @author Vladimir Hromada
 */
public interface CategoryRepository extends JpaRepository<Category, Integer> {
}

File: Bookcase core/src/main/kotlin/cz/vhromada/book/repository/CategoryRepository.kt
package cz.vhromada.book.repository

import cz.vhromada.book.domain.Category
import org.springframework.data.jpa.repository.JpaRepository

/**
 * An interface represents repository for categories.
 *
 * @author Vladimir Hromada
 */
interface CategoryRepository : JpaRepository<Category, Int>
